cms/templates/product_page.html

```diff
@@ -24,17 +24,19 @@ <h1>{{ page.title }}</h1>
                 <div id="productDetailEnrollment"></div>
               </div>
             </div>
-            {% if page.video_url %}
-              <div class="content-col video">
-                <video-js
-                  class="vjs-default-skin vjs-big-play-centered" 
-                  controls 
-                  data-setup="{{page.video_player_config}}"
-                  preload='metadata'
-                >
-                </video-js>
-              </div>
-            {% endif %}
+            <div class="content-col video">
```

Comment:
Now that we are not only dealing with video so it would make sense to name the `video` class accordingly, probably you can name it `feature-media`or `hero-media` or any name you'd like.